jobboard/migrations/0002_fixed_data.py

```diff
@@ -0,0 +1,37 @@
+from django.db import migrations, transaction
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ('jobboard', '0001_initial'),
+    ]
+
+    def generate_data(apps, schema_editor):
+        from jobboard.models import Region, JobTitleKeyword
+
+        region_names = [
+            "Central",
+            "South",
+            "North",
+            "Tel Aviv",
+            "Haifa",
+            "Jerusalem"
+        ]
+
+        keywords = [
+            "software",
+            "engineer",
```

Comment:
isn't it better to have it as a one keyword?

Reply:
I don't think I understand what you mean